hummingbot/strategy/uniswap_v3_lp/uniswap_v3_lp.py

```diff
@@ -351,8 +351,28 @@ def execute_proposal(self, proposal):
 
     def range_position_remover(self):
         """
-        This function removes  positions that are out of range and have attained min profitability.
+        This removes the farthest position.
+        """
+        inactive_sells = [position for position in self.active_positions if position.lower_price > self._last_price]
+        inactive_buys = [position for position in self.active_positions if position.upper_price < self._last_price]
+        farthest_position = None
+        if len(inactive_sells) > 1:
+            for position in inactive_sells:
+                farthest_position = position if farthest_position is None else position
```

Comment:
This `if` expression doesn't make sense as it always returns position regardless.